src/app/features/right-pane/versions-section.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react"
+import VersionItem from "./version-item"
+import {useSectionTreeDefaults} from "../sidebar/hooks"
+import {Tree} from "react-arborist"
+import {useSelector} from "react-redux"
+import Current from "src/js/state/Current"
+import styled from "styled-components"
+
+const SectionWrapper = styled.div`
+  flex: 1;
+  margin-top: 8px;
+`
+
+const VersionsSection = () => {
+  const {resizeRef, defaults} = useSectionTreeDefaults()
+  const query = useSelector(Current.getQuery)
+
+  return (
+    <SectionWrapper ref={resizeRef}>
+      <Tree
```

Comment:
Are these nodes draggable?

Reply:
It should probably be disabled.

Reply:
Oh, actually defaults already do disabledrag